drawing.s43

```diff
@@ -43,48 +43,92 @@ displaybattery:
       jeq displayFifthSegment
       cmp #6, r12
       jeq displaySixthSegment
+      pop r12
       ret
       
 displayOutline:
-      MOV.B   #00010000B, &0X0a31  ; []
-      MOV.B   #00010000B, &0X0a2d  ; []]
+      BIS.B   #00010000B, &0X0a31  ; []
       ret
  
 displayfirstSegment:
       MOV.B   #00110000B, &0X0a31 
-      MOV.B   #00010000B, &0X0a2d  
+      MOV.B   #00000000B, &0X0a2d  
       ret
       
 displaySecondSegment:
       MOV.B   #00110000B, &0X0a31  
-      MOV.B   #00110000B, &0X0a2d  
+      MOV.B   #00100000B, &0X0a2d  
```

Comment:
I think you forgot the other segments, to change them to bic instead of mov

Reply:
you mean BIS?